plugins/parsers/csv/README.md

```diff
@@ -33,8 +33,35 @@ values.
   ## If this is not specified, type conversion will be done on the types above.
   csv_column_types = []
 
-  ## Indicates the number of rows to skip before looking for header information.
+  ## Indicates the number of rows to skip before looking for metadata and header information.
   csv_skip_rows = 0
+  
+  
+  ## Indicates the number of rows to parse as metadata before looking for header information. 
+  ## By default, the parser assumes there are no metadata rows to parse. 
+  ## If set, the parser would use the provided separators in the cvs_metadata_separators to look for metadata.
+  csv_metadata_rows = 0
+  
+  ## A list of metadata separators. If csv_metadata_rows is set,
+  ## cvs_metadata_separators must contain at least one separator.
+  ## Please note that separators are case sensitive.
+  ## For example:
+  ## cvs_metadata_separators = [",", ":", "="]
+  ## to parse "Key1,Value1" "Key2:Value2" "Key3=Value3"
+  ## the returned (key,value) pares would be as follows:
+  ## (Key1,Value1), (Key2,Value2), (Key3,Value3)
```

Comment:
This section confuses me a bit. I think you better write the resulting tags/fields in influx format.

Reply:
Good idea. I will do that tomorrow.

Reply:
You also might want to add here (or in the section for `metadata_rows`) that the key value pairs will be added as field by default and they can use `tag_columns` to convert them to tags.

Reply:
Done